source/common/http/http3/conn_pool.h

```diff
@@ -26,6 +37,41 @@ class ActiveClient : public MultiplexedActiveClientBase {
                                     data) {}
 };
 
+// Http3 subclass of FixedHttpConnPoolImpl which exists to store quic data.
+class Http3ConnPoolImpl : public FixedHttpConnPoolImpl {
+public:
+  Http3ConnPoolImpl(Upstream::HostConstSharedPtr host, Upstream::ResourcePriority priority,
+                    Event::Dispatcher& dispatcher,
+                    const Network::ConnectionSocket::OptionsSharedPtr& options,
+                    const Network::TransportSocketOptionsSharedPtr& transport_socket_options,
+                    Random::RandomGenerator& random_generator,
+                    Upstream::ClusterConnectivityState& state, CreateClientFn client_fn,
+                    CreateCodecFn codec_fn, std::vector<Http::Protocol> protocol,
+                    TimeSource& time_source)
+      : FixedHttpConnPoolImpl(host, priority, dispatcher, options, transport_socket_options,
```

Comment:
Your call, but I'd be inclined to move the body of both the constructor and destructor to the .cc file.

Reply:
done